src/System.Private.CoreLib/shared/System/Decimal.cs

```diff
@@ -617,7 +617,7 @@ private static decimal Round(ref decimal d, int decimals, MidpointRounding mode)
         {
             if ((uint)decimals > 28)
                 throw new ArgumentOutOfRangeException(nameof(decimals), SR.ArgumentOutOfRange_DecimalRound);
-            if ((uint)mode > (uint)MidpointRounding.AwayFromZero)
+            if ((uint)mode > (uint)MidpointRounding.ToPositiveInfinity)
```

Comment:
`AwayFromZero` and `ToPositiveInfinity` are not the same and are not interchangeable.

Reply:
nevermind, this is the ArgumentException check, I misread the code.